From 4db206272e98143c2a83af7b67248ed5b556d6d9 Mon Sep 17 00:00:00 2001
From: Etienne Carriere <etienne.carriere@st.com>
Date: Tue, 12 May 2020 13:56:24 +0200
Subject: [PATCH] xtest: regression 1027,1028: skip test

Skip regression tests 1027 and 1028 that mandate support for UUIDv5
in Linux kernel OP-TEE driver.

Signed-off-by: Etienne Carriere <etienne.carriere@st.com>
---
 host/xtest/regression_1000.c | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/host/xtest/regression_1000.c b/host/xtest/regression_1000.c
index 8142f37..f789247 100644
--- a/host/xtest/regression_1000.c
+++ b/host/xtest/regression_1000.c
@@ -2094,6 +2094,11 @@ static void xtest_tee_test_1027(ADBG_Case_t *c)
 	TEEC_UUID uuid_ns = { };
 	char uuid_name[TEE_UUID_NS_NAME_SIZE] = { };
 
+	if (c) {
+		Do_ADBG_Log("Skip test 1028: needs change in Linux kernel");
+		return;
+	}
+
 	result = xtest_uuid_from_str(&uuid_ns, client_uuid_linux_ns);
 
 	if (!ADBG_EXPECT_TEEC_SUCCESS(c, result))
@@ -2142,6 +2147,11 @@ static void xtest_tee_test_1028(ADBG_Case_t *c)
 	char uuid_name[TEE_UUID_NS_NAME_SIZE] = { };
 	uint32_t group = 0;
 
+	if (c) {
+		Do_ADBG_Log("Skip test 1028: needs change in Linux kernel");
+		return;
+	}
+
 	group = getegid();
 
 	result = xtest_uuid_from_str(&uuid_ns, client_uuid_linux_ns);
-- 
2.17.1

